testbook/client.py

```diff
@@ -1,32 +1,37 @@
 from inspect import getsource
 from textwrap import dedent
+from typing import Any, Dict, List, Optional, Union
 
 from nbclient import NotebookClient
 from nbclient.exceptions import CellExecutionError
 from nbformat.v4 import new_code_cell
 
 from .exceptions import (
     TestbookCellTagNotFoundError,
-    TestbookSerializeError,
     TestbookExecuteResultNotFoundError,
+    TestbookSerializeError,
 )
-from .utils import random_varname
+from .reference import TestbookObjectReference
 from .testbooknode import TestbookNode
 from .translators import PythonTranslator
-from .reference import TestbookObjectReference
+from .utils import random_varname
 
 
 class TestbookNotebookClient(NotebookClient):
     def __init__(self, nb, km=None, **kw):
         super().__init__(nb, km=km, **kw)
 
-    def ref(self, name):
+    def ref(self, name: str) -> TestbookObjectReference:
+        """
+        Return a reference to an object in the kernel
+        """
+
         # Check if exists
         self.inject(name)
         return TestbookObjectReference(self, name)
 
     @staticmethod
-    def _construct_call_code(func_name, args=None, kwargs=None):
+    def _construct_call_code(func_name: str, args=None, kwargs=None) -> str:
```

Comment:
I think you can also type args as `Optional[List]` and kwargs as `Optional[Dict]`